Java/main-sample/src/main/java/com/perfectomobile/appTest/MyApp.java

```diff
@@ -185,7 +188,7 @@ public static void main(String[] args) throws IOException {
 
             driver.quit();
         }
-
+        Assert.assertEquals(testpassed,true);
         System.out.println("Run ended");
```

Comment:
Why do we care if it passed or failed as long as it run. It's a code sample, even if there is no device available.

Reply:
its betetr to have some indication that somthing is working there. it might be filled with exceptions, we will never know. from cli build always passed.